src/jit/emitxarch.cpp

```diff
@@ -11589,8 +11589,8 @@ BYTE* emitter::emitOutputLJ(BYTE* dst, instrDesc* i)
             assert(INS_ja  + (INS_l_jmp - INS_jmp) == INS_l_ja);
             assert(INS_js  + (INS_l_jmp - INS_jmp) == INS_l_js);
             assert(INS_jns + (INS_l_jmp - INS_jmp) == INS_l_jns);
-            assert(INS_jpe + (INS_l_jmp - INS_jmp) == INS_l_jpe);
-            assert(INS_jpo + (INS_l_jmp - INS_jmp) == INS_l_jpo);
+            assert(INS_jp  + (INS_l_jmp - INS_jmp) == INS_l_jp);
+            assert(INS_jnp + (INS_l_jmp - INS_jmp) == INS_l_jnp);
```

Comment:
The Intel arch does define both jp and jpe,  I don't think that we even use these instructions but it seems to me that jpe and jpo are more understandable than jp and jnp

https://c9x.me/x86/html/file_module_x86_id_146.html

Reply:
IMO jpe/jpo are more understandable when used for the original purpose - parity checking. But that's history now and the only purpose of the PF flag these days is detecting "unordered" floating point comparisons. In this case jp/jnp seem preferable because the clearly indicate that the branch is taken when PF=1/0.

Reply:
VS's own disassembler also uses jp/jnp:
```
00007FFD026C049F C4 E1 7B 10 05 E8 42 EF FF vmovsd      xmm0,qword ptr [7FFD025B4790h]  
00007FFD026C04A8 C4 E1 79 2E 05 E7 42 EF FF vucomisd    xmm0,mmword ptr [7FFD025B4798h]  
00007FFD026C04B1 7A 02                jp          00007FFD026C04B5  
00007FFD026C04B3 74 0B                je          00007FFD026C04C0  
```

Reply:
OK I am good with this change.
Thanks for the explaination.